testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpClientTest.groovy

```diff
@@ -1139,11 +495,34 @@ abstract class HttpClientTest<REQUEST> extends InstrumentationSpecification {
     return true
   }
 
-  URI removeFragment(URI uri) {
-    return new URI(uri.scheme, null, uri.host, uri.port, uri.path, uri.query, null)
+  Throwable clientSpanError(URI uri, Throwable exception) {
+    return exception
+  }
+
+  // parent span must be cast otherwise it breaks debugging classloading (junit loads it early)
```

Comment:
Is this comment still valid? Now that we have a separate testing java agent there should be no problem with loading that stuff early (probably...)